pkg/authentication/handlers.go

```diff
@@ -258,7 +258,7 @@ func (a *API) handleLogout(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *API) uiRedirect(w http.ResponseWriter, r *http.Request, nextTo string) {
-	redirect := ui.UIPrefix
+	redirect := a.contextPath + ui.UIPrefix
```

Comment:
we should make sure this is a valid uri, can we use http lib to parse and join?

Reply:
Damn, that shouldn't be in here. Good catch